src/toil/wdl/wdltoil.py

```diff
@@ -708,6 +705,94 @@ def is_url(filename: str, schemes: List[str] = ['http:', 'https:', 's3:', 'gs:',
                 return True
         return False
 
+def import_url_files(environment: WDLBindings, file_source: Union[AbstractFileStore, Toil], path: Optional[List[str]] = None, include_self: bool = False) -> Optional[Dict[str, str]]:
+    """
+    Iterate through the environment and import all files from the possible URLs. This will not import if the file is detected as local.
+    Returns a mapping of the original file names to virtualized URIs
+    :param environment: Bindings to evaluate on
+    :param file_source: Context to upload/virtualize files with
+    :param path: List of paths to search through
+    :return:
+    """
+    parent_to_id: Dict[str, uuid.UUID] = {}
+    filename_to_uri: Dict[str, str] = {}
+    def import_file_from_url(filename: str) -> str:
+        """
+        Import a file if from a URL and return a virtualized filename for it.
+        """
+        # Search through any input search paths passed in and download it if found
+        tried = []
+        for candidate_uri in potential_absolute_uris(filename, path if path is not None else []):
+            if not include_self and candidate_uri == filename:
+                # If the candidate uri is unchanged, as in is equal to the original uri
+                # we can discard it if unwanted; for example, miniwdl's backend can handle URLs fine,
+                # and carrying through the url through the workflow won't actually hit the imported file
+                # without more machinery
```

Comment:
I don't understand this comment. What does it mean to "hit" the file? What does it mean for the file to be "unwanted"?

Reply:
I changed the implementation so this functionality doesn't exist anymore